app/experimenter/static/js/components/TimelinePopForm.js

```diff
@@ -152,6 +159,11 @@ class TimelinePopForm extends React.PureComponent {
     return choicesJSX;
   }
 
+  isWindowsVersionEnabled() {
+    const platforms = this.state.data.get("platforms").toJS();
+    return platforms.length === 1 && platforms[0].value === "All Windows";
```

Comment:
Yep this looks good I would just move the actual `"All Windows"` into a string constant in `constants.js`